Gaijinizer patch/Map061.txt

```diff
@@ -138,48 +151,53 @@ No, I'm sure E-003 will get lost ....#MTLed
 ただでさえ劣勢な人間たちに、俺たちはみすみすヤバい脅威を
 新たに送り込むことになっちまうんだぜ…？
 > CONTEXT: Map061/events/5/pages/0/93/Dialogue/other2
-But what happens then? Even to the inferior
-people, we helplessly. What do you want to do?#MTLed
+But what happens then?
+
+At best, we've just knowingly sent in a dangerous
+new threat against humanity...#Ron
 > END STRING
 
 > BEGIN STRING
 極めつけは、E-003の再生能力が異常すぎて
 
 肝心の『安全装置』すら効かねぇって話じゃねぇか…
 > CONTEXT: Map061/events/5/pages/0/100/Dialogue/other2
-At the very end, the playback ability of E-003 is
-too abnormal. Even if the important "safety
-device" is effective, it is not talk about it ...#MTLed
+And the cherry on top is E-003's abnormally
+powerful regenerative ability. All the "compliance
+devices" in the world ain't gonna be able
+to keep it down...#Ron
 > END STRING
 
 > BEGIN STRING
 そんな奴を野に放つような命令を平然とやるんだ…！
 
 ホント、どうかしてるぜ……！この組織はよぉ…！！
 > CONTEXT: Map061/events/5/pages/0/104/Dialogue/other2
-I'm going to make an order to release such a man
-in the field ...! Really, I'm sorry ...! This
-organization is ...! !#MTLed
+They're orderin' us to release that kind of thing
+into the wild...!!
+
+What the hell is wrong with this organisation...?!#Ron
 > END STRING
 
 > BEGIN STRING
 ………今更だろう。
 
 このM.O.Wが正気の沙汰ではないことぐらい。
 > CONTEXT: Map061/events/5/pages/0/108/Dialogue/other2
-... ...... I think it's better now.
-It is about that this M.O.W. is not a sane
-person.#MTLed
+... It's too late for any of that now.
+
+M.O.W. has already lost its mind.#Ron
 > END STRING
 
 > BEGIN STRING
 お前は金のため、俺は恩返しのため。
 
 あえてこの組織で働くと、最初に互いに語りあったはずだ。
 > CONTEXT: Map061/events/5/pages/0/112/Dialogue/other2
-You are for money, I for giving back.
-If you dare to work in this organization, you
-should have first talked to each other.#MTLed
+You're here for the money, I'm here as a favor.
+
+If you've gone out of your way to work here, you
+must've talked to them about it first.#RonIffy
```

Comment:
For the first line, I'd say:
>_**You're here for the money, and I'm here because
I owe them one. (Continued...)**_

From the impression I'm getting from the JP, I feel like they're saying something
to the effect of:
"When you signed up with M.O.W, you should have known
you would be doing something like this."